syncer/syncer.go

```diff
@@ -998,6 +998,7 @@ func (s *Syncer) sync(tctx *tcontext.Context, queueBucket string, db *DBConn, jo
 	for {
 		select {
 		case sqlJob, ok := <-jobChan:
+			queueSizeGauge.WithLabelValues(s.cfg.Name, queueBucket).Set(float64(len(jobChan)))
```

Comment:
multiple queues share same bucket, it would be problem i under some cases. 
It is recommended that both slow query
 threshold and bucket count be configurable, and these can be optimized later